packages/ui/lib/components/HeaderNav/HeaderNav.tsx

```diff
@@ -309,7 +311,12 @@ const NavSectionMobile = ({
     return (
       <div className="flex flex-col gap-4">
         {title?.length > 0 && (
-          <Title title={title} open={open} onClick={() => setOpen(!open)} />
+          <Title
+            title={title}
```

Comment:
nit: Can we re-name `title` prop to be text or content? Something which we can follow consistently. We use label but that doesn't seen appropriate here. WDYT?

Reply:
renamed! better then `title` :D